tests/multi_ptr/multi_ptr_subscript_op_common.h

```diff
@@ -98,7 +98,7 @@ class run_subscript_op_tests {
         if constexpr (space == sycl::access::address_space::local_space) {
           sycl::local_accessor<T> exp_arr_acc{{array_size}, cgh};
           sycl::local_accessor<T> exp_arr_neg_acc{{array_size}, cgh};
-          cgh.single_task([=] {
+          cgh.parallel_for(sycl::nd_range<1>({1}, {1}), [=](auto item) {
```

Comment:
Why do we need this change?

Reply:
According to the [spec](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#sec:accessor.local), `local_accessor` is only allowed in `nd_range` `parallel_for`:
> A local accessor must not be used in a [SYCL kernel function](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#sycl-kernel-function) that is invoked via single_task or via the simple form of parallel_for that takes a range parameter. In these cases submitting the kernel to a queue must throw a synchronous exception with the errc::kernel_argument error code.

Reply:
Where is the comment in the code explaining this? :-)